katdal/datasources.py

```diff
@@ -518,6 +521,10 @@ class TelstateDataSource(DataSource):
     ----------
     telstate : :class:`katsdptelstate.TelescopeState` object
         Telescope state with appropriate views
+    capture_block_id : string
+        Capture block ID
+    stream_name : string
+        Name of the L0 stream
```

Comment:
Would this necessarily be L0 (just checking)?

Reply:
It's the same argument passed to (or returned from) `view_l0_capture_stream`, so I assume so.